crates/proof-of-sql/benches/scaffold/querys.rs

```diff
@@ -118,6 +118,27 @@ const LARGE_COLUMN_SET_COLUMNS: &[(&str, ColumnType, OptionalRandBound)] = &[
     ("g", ColumnType::VarChar, None),
     ("h", ColumnType::Scalar, None),
 ];
+const COMPLEX_CONDITION_TITLE: &str = "Complex Condition";
+const COMPLEX_CONDITION_SQL: &str =
+    "SELECT * FROM table WHERE (a > c * c AND b < c + 10) OR (d = 'abc')";
```

Comment:
NIT: Same thing as before.

Reply:
`abc` follows the same motivation as the last comment.